### DISCUSSION TITLE
clone an environment not a project

### DISCUSSION THREAD

**Discussion by Geczy** [coollabsio/coolify#1582] [Feature Requests]
i want to clone an environment not a project

this cloned the env into a new project not same proj. bug?

![image](https://github.com/coollabsio/coolify/assets/1036968/30772a39-48fc-4d6d-bb1f-57e0326efa3c)

**Comment by PabloDinella** [comment]
Answering for future readers like me:

You have the option to clone into a new project or just clone as a new environment. I found the UX for cloning a bit confusing but it actually worked.

![image](https://github.com/user-attachments/assets/35bd0a47-754d-42c5-a8d5-2b3e42ca8b29)

**Comment by RatulMaharaj** [comment]
Also found the UX around cloning to be a bit confusing. It would be really great if we could clone one resource from an environment (e.g. dev) into another one (e.g. prod) without needing to clone all resources.
